lib/ufe/Global.cpp

```diff
@@ -120,6 +123,11 @@ MStatus initialize()
 
 	g_StagesSubject = StagesSubject::create();
 
+    // Register for UFE string to path service using path component separator '/'
+#if UFE_PREVIEW_VERSION_NUM >= 2011
+    UFE_V2(Ufe::PathString::registerPathComponentSeparator(g_USDRtid, '/');)
```

Comment:
Register with optional UFE path string functionality.  In USD, the path component separator is forward slash '/'.